src/transformStateWithClones.js

```diff
@@ -7,7 +7,53 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrayObjects = [];
+  let currentState = {};
+
+  for (const object of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
object слишком общее название